bin/ream/src/main.rs

```diff
@@ -53,12 +53,15 @@ async fn main() {
                 total_peers: 0,
             };
 
-            let _ream_db = ReamDB::new(config.data_dir, config.ephemeral)
-                .expect("unable to init Ream Database");
+            let ream_db = Arc::new(
+                ReamDB::new(config.data_dir, config.ephemeral)
+                    .expect("unable to init Ream Database"),
+            );
```

Comment:
this shouldn't be wrapped in an arc, the field db inside already is so this arc isn't doing anything